twigs.cc

```diff
@@ -228,6 +239,7 @@ void SystemInit() {
   TCCR1A = 0;
   TCCR1B = 5;
 
+  ClockInit();
```

Comment:
This might be a little nitpicky but unless there's reason not to, I would add this to `SystemInit` after the call to `GateInputsInit()` .  That would also eliminate the need for the function declaration above

Reply:
I fixed.